.travis.yml

```diff
@@ -17,6 +17,15 @@ php:
 
 matrix:
   fast_finish: true
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION=2.6.*
```

Comment:
You still need to include this environment variable in the `install` step. Now it just always includes the symfony version set in `composer.json`. Should become something like:

```
install:
  - composer require symfony/symfony:${SYMFONY_VERSION}
  - composer install
```